### PR TITLE
Ddpb 3480 - Add the money out short behat section tests

### PR DESCRIPTION
**Pull request by jamesrwarren**
## Purpose
Add the money out short behat section tests.

image

Fixes DDPB-3480

## Approach
Had to create some different report type fixtures. You may notice I didn't totally rename these on the client side. This is for two reasons. Firstly wasn't sure how these would now fit into the new API structure so no point having to restest everything and secondly when i tried and tested some of the old tests they had some other issues not related to this that I didn't want to have to fix as part of this ticket as I know we wanted to clear up our board so we can get Alex's aforementioned mega restructure ticket in.

## Checklist
- [x] I have performed a self-review of my own code
- [ ] I have updated documentation (Confluence/ADR/tech debt doc) where relevant
- [ ] I have added tests to prove my work
- [ ] The product team have approved these changes

### Frontend
- [ ] I have run an in-browser accessibility test (e.g. WAVE, Lighthouse)
- [ ] There are no deprecated CSS classes noted in the profiler
- [ ] Translations are used and the profiler doesn't identify any missing
- [ ] Any links or buttons added are screen reader friendly and contextually complete
- [ ] If adding GA events, I have updated or [checked](docs/runbooks/GOOGLE-ANALYTICS.md) the existing category or label values

### CODE REVIEW

File: behat/tests/bootstrap/v2/reporting/sections/DocumentsSectionTrait.php [hunk 1]
```diff
@@ -94,11 +96,7 @@ public function theDocumentsSummaryPageShouldContainDocumentsIUploaded()
             );
         }
 
-        $descriptionLists = $this->getSession()->getPage()->findAll('css', 'dl');
-
-        if (count($descriptionLists) === 0) {
-            $this->throwContextualException('A dl element was not found on the page - make sure the current url is as expected');
-        }
+        $descriptionLists = $this->findAllCssElements('dl');
```
Comment (acsauk) [inline behat/tests/bootstrap/v2/reporting/sections/DocumentsSectionTrait.php:99]: 😎

File: behat/tests/bootstrap/v2/reporting/sections/MoneyOutShortSectionTrait.php [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+                $oneOffPaymentRow->getHtml(),
+                'Short Money Out One Off Payments Items'
+            );
+            $this->assertStringContainsString($amountFormatted, $oneOffPaymentRow->getHtml(), 'Short Money Out One Off Payments Items');
+            $this->assertStringContainsString($date, $oneOffPaymentRow->getHtml(), 'Short Money Out One Off Payments Items');
+
+            $this->oneOffPaymentsTotal += floatval($this->oneOffPaymentsList[$oneOffPaymentRowKey]['amount']);
+        }
+
+        $this->assertStringContainsString($this->moneyFormat($this->oneOffPaymentsTotal), $oneOffPaymentTotalRow->getHtml(), 'Short Money Out One Off Payments Total');
```
Comment (acsauk) [inline behat/tests/bootstrap/v2/reporting/sections/MoneyOutShortSectionTrait.php:330]: May have missed it but are we asserting on the description of the payment here?

Comment (jamesrwarren) [reply behat/tests/bootstrap/v2/reporting/sections/MoneyOutShortSectionTrait.php:330]: yeah it just got formatted a bit funny cos it's a long line: $this->oneOffPaymentsList[$oneOffPaymentRowKey]['description']

**Review by acsauk — COMMENTED** [review]
Great stuff - one Q on if we're asserting on descriptions in the summary page.

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**